.docker/src-cron/etc/periodic/5min/run-lightweight-cron.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# grab the cloudgov space we are hosted in
+APP_SPACE=$(echo "$VCAP_APPLICATION" | jq -r '.space_name')
+SECRETS=$(echo $VCAP_SERVICES | jq -r '.["user-provided"][] | select(.name == "cron-secrets") | .credentials')
+CRON_KEY=$(echo $SECRETS | jq -r '.CRON_KEY')
+
+# only the 1st instance within cloud formation should actually do anything on cron
+if [ "${CF_INSTANCE_INDEX:-''}" == "0" ]; then
+
+  # Use unique uri per environment - default to prod
+  if [ "${APP_SPACE}" = "local" ]; then
+    URI="http://cms-usagov.docker.local"
+  else
+    URI="http://cms-${APP_SPACE}-usagov.apps.internal"
+  fi
+
+  echo "Running lightweight cron"
+  curl "$URI/scheduler/cron/$CRON_KEY"
```

Comment:
Spews too much to the log files (especially when the CRON_KEY isn't set up on the other side.